site/topics/references/references.rst

```diff
@@ -0,0 +1,516 @@
+******************
```

Comment:
I assume this is like #214 where this is copy/paste. I have not reviewed the content closely.

Reply:
Bingo. 

In future PRs where this happens, I will make it more explicit that it is NOT a content change.